### PR TITLE
Add ToSC Volume 2016 papers

### PR DESCRIPTION
**Pull request by pfasante**
This PR contains ToSC Volume 2016 Issue 1 and 2 papers.
New abbreviations for ToSC are added to abbrev.bibyml.

TODO:
We should decide on how to handle the URL fields. I give a suggestion in #23.
I accidentally branched from my master branch that already contained the url
fields, so at the moment this PR contains that changes to.
If needed I can revert that, but preferable I would avoid double work.

### CODE REVIEW

File: crypto_db.bib [hunk 1]
```diff
@@ -371908,145 +371908,368 @@ @Misc{EPRINT:BluDeSSti96
...
+  pages =        "1--29",
+  year =         "2016",
+  url =          "http://tosc.iacr.org/index.php/ToSC/article/view/563",
+  doi =          "10.13154/tosc.v2016.i2.1-29",
+}
+
+@Article{ToSC:BhaNan16,
+  author =       "Ritam Bhaumik and
+                  Mridul Nandi",
+  title =        "OleF: an Inverse-Free Online Cipher. An Online {SPRP} with an Optimal Inverse-Free Construction",
```
Comment (tlepoint) [inline crypto_db.bib]: Should be "{OleF}"

File: crypto_db.bib [hunk 2]
```diff
@@ -371908,145 +371908,368 @@ @Misc{EPRINT:BluDeSSti96
...
+  pages =        "80--98",
+  year =         "2016",
+  url =          "http://tosc.iacr.org/index.php/ToSC/article/view/566",
+  doi =          "10.13154/tosc.v2016.i2.80-98",
+}
+
+@Article{ToSC:PerUdo16,
+  author =       "L{\'{e}}o Perrin and
+                  Aleksei Udovenko",
+  title =        "Exponential S-Boxes: a Link Between the S-Boxes of BelT and Kuznyechik/Streebog",
```
Comment (tlepoint) [inline crypto_db.bib]: Correct capital letters

File: crypto_db.bib [hunk 3]
```diff
@@ -371908,145 +371908,368 @@ @Misc{EPRINT:BluDeSSti96
...
+  year =         "2016",
+  url =          "http://tosc.iacr.org/index.php/ToSC/article/view/571",
+  doi =          "10.13154/tosc.v2016.i2.192-225",
+}
+
+@Article{ToSC:BirKhoPer16,
+  author =       "Alex Biryukov and
+                  Dmitry Khovratovich and
+                  L{\'{e}}o Perrin",
+  title =        "Multiset-Algebraic Cryptanalysis of Reduced Kuznyechik, Khazad, and secret SPNs",
```
Comment (tlepoint) [inline crypto_db.bib]: Correct capital letters

File: crypto_db.bib [hunk 4]
```diff
@@ -371908,145 +371908,368 @@ @Misc{EPRINT:BluDeSSti96
...
+  pages =        "248--260",
+  year =         "2016",
+  url =          "http://tosc.iacr.org/index.php/ToSC/article/view/573",
+  doi =          "10.13154/tosc.v2016.i2.248-260",
+}
+
+@Article{ToSC:ChoMai16,
+  author =       "Arka Rai Choudhuri and
+                  Subhamoy Maitra",
+  title =        "Significantly Improved Multi-bit Differentials for Reduced Round Salsa and ChaCha",
```
Comment (tlepoint) [inline crypto_db.bib]: Correct capital letters

File: crypto_db.bib [hunk 5]
```diff
@@ -371908,145 +371908,368 @@ @Misc{EPRINT:BluDeSSti96
...
-  volume    = 2016,
-  number    = 1,
-  pages     = "158--176",
-  year      = 2016,
-  url       = "http://tosc.iacr.org/index.php/ToSC/article/view/540",
-  doi       = "10.13154/tosc.v2016.i1.158-176",
+  author =       "Atul Luykx and
+                  Bart Mennink and
+                  Samuel Neves",
+  title =        "Security Analysis of BLAKE2's Modes of Operation",
```
Comment (tlepoint) [inline crypto_db.bib]: Correct capital letters

File: crypto_db.bib [hunk 6]
```diff
@@ -371908,145 +371908,368 @@ @Misc{EPRINT:BluDeSSti96
...
-  publisher =   toscpub,
-  volume =      2016,
-  number =      1,
-  pages =       "95--113",
-  year =        2016,
-  url =         "http://tosc.iacr.org/index.php/ToSC/article/view/537",
-  doi =         "10.13154/tosc.v2016.i1.95-113",
+  author =       "Sumanta Sarkar and
+                  Habeeb Syed",
+  title =        "Lightweight Diffusion Layer: Importance of Toeplitz Matrices",
```
Comment (tlepoint) [inline crypto_db.bib]: Correct capital letters

File: crypto_db.bib [hunk 7]
```diff
@@ -371908,145 +371908,368 @@ @Misc{EPRINT:BluDeSSti96
...
-  year =        2016,
-  url =         "http://tosc.iacr.org/index.php/ToSC/article/view/534",
-  doi =         "10.13154/tosc.v2016.i1.33-56",
+  author =       "Jian Guo and
+                  J{\'{e}}r{\'{e}}my Jean and
+                  Ivica Nikolic and
+                  Kexin Qiao and
+                  Yu Sasaki and
+                  Siang Meng Sim",
+  title =        "Invariant Subspace Attack Against Midori64 and The Resistance Criteria for S-box Designs",
```
Comment (tlepoint) [inline crypto_db.bib]: Correct capital letters

File: crypto_db.bib [hunk 8]
```diff
@@ -371908,145 +371908,368 @@ @Misc{EPRINT:BluDeSSti96
...
-  publisher =   toscpub,
-  volume =      2016,
-  number =      1,
-  pages =       "13--32",
-  year =        2016,
-  url =         "http://tosc.iacr.org/index.php/ToSC/article/view/532",
-  doi =         "10.13154/tosc.v2016.i1.13-32",
+  author =       "Xiaoyang Dong and
+                  Xiaoyun Wang",
+  title =        "Chosen-Key Distinguishers on 12-Round Feistel-SP and 11-Round Collision Attacks on Its Hashing Modes",
```
Comment (tlepoint) [inline crypto_db.bib]: Correct capital letters

File: crypto_db.bib [hunk 9]
```diff
@@ -371908,145 +371908,368 @@ @Misc{EPRINT:BluDeSSti96
...
-  issn =        toscissn,
-  publisher =   toscpub,
-  volume =      2016,
-  number =      1,
-  pages =       "1--12",
-  year =        2016,
-  url =         "http://tosc.iacr.org/index.php/ToSC/article/view/531",
-  doi =         "10.13154/tosc.v2016.i1.1-12",
+  author =       "J{\'{e}}r{\'{e}}my Jean",
+  title =        "Cryptanalysis of Haraka",
```
Comment (tlepoint) [inline crypto_db.bib]: Correct capital letters?

File: crypto_db.bib [hunk 10]
```diff
@@ -371908,145 +371908,368 @@ @Misc{EPRINT:BluDeSSti96
...
+  url =          "http://tosc.iacr.org/index.php/ToSC/article/view/575",
+  doi =          "10.13154/tosc.v2016.i2.288-306",
+}
+
+@Article{ToSC:GLNS16,
+  author =       "Jian Guo and
+                  J{\'{e}}r{\'{e}}my Jean and
+                  Ivica Nikolic and
+                  Yu Sasaki",
+  title =        "Meet-in-the-Middle Attacks on Classes of Contracting and Expanding Feistel Constructions",
```
Comment (tlepoint) [inline crypto_db.bib]: Correct capital letters

File: crypto_db.bib [hunk 11]
```diff
@@ -371908,145 +371908,368 @@ @Misc{EPRINT:BluDeSSti96
...
+  url =          "http://tosc.iacr.org/index.php/ToSC/article/view/574",
+  doi =          "10.13154/tosc.v2016.i2.261-287",
+}
+
+@Article{ToSC:GWLZ16,
+  author =       "Zhiyuan Guo and
+                  Wenling Wu and
+                  Renzhang Liu and
+                  Liting Zhang",
+  title =        "Multi-key Analysis of Tweakable Even-Mansour with Applications to Minalpher and {OPP}",
```
Comment (tlepoint) [inline crypto_db.bib]: Correct capital letters

**Comment by pfasante** [comment]
Oh yes, of course. I totally forgot that.
Is the new commit ok, regarding upper/lower case?

**Comment by pfasante** [comment]
I removed the URL field change from this pr, because it does not belong here.

**Comment by pfasante** [comment]
we now also have the v2017 i1 entries ready

**Comment by fabrice102** [comment]
Manually merged.

**Pull request closed without merging**